integration-tests/typings/embrace.d.ts

```diff
@@ -14,4 +14,17 @@ interface EmbraceRequestBody {
   json: EmbracePayload;
 }
 
-export type {EmbraceRequestBody, SessionPayload};
+interface Data {
+  Body: string;
+}
+interface EmbraceData {
+  Spans: IData[];
```

Comment:
we should be good with just `Spans` and `Logs` here, the rest are part of the v1/ endpoints and no longer used

Reply:
ok